### ISSUE TITLE
BatchNorm PointSegDA

### ISSUE THREAD

**Issue by antoalli** [IdanAchituve/DefRec_and_PCM#6]
Hi! from the network implementation (PointSegDA experiments) I see that you pruned many BatchNorm layers which were in the original pytorch dgcnn implementation, for example in the main feature extractor 'shared_layers'. Is there any specific reason to do so? Have you found that removing BN layers the results are more stable across domains?
Thanks a lot

**Comment by IdanAchituve** [comment]
Hi Antoalli,
I removed the BN layers because I wanted to neutralize its effect in bridging the domain gap.
I didn't experiment with that much. I believe that adding these layers back will not make a big difference.

Cheers,
Idan

**Comment by antoalli** [comment]
Seems resonable to me, as last question for the segmentation experiments can I assume that the training parameters used are the default parsed arguments? For example Adam optimizer with lr 1e-3 and so on?
I forgot to mention in my first message but thank you for the very interesting work,
Antonio

**Comment by IdanAchituve** [comment]
Thanks. No problem.
Yes, the default parameters yielded the best results in most cases. However, please note that in the paper we applied a grid search per adaptation setup, so the results with these parameters may not match exactly to those published.
